practice/1_python_part_1/task6.py

```diff
@@ -20,5 +20,13 @@
 
 
 def get_min_max(filename: str) -> Tuple[int, int]:
-    ...
-
+  l = []
+  with open(filename) as opened_file:
+    for line in opened_file:
+      l.append(line.replace('\n',''))
+  for digit in l:
+    if digit.isdigit() or digit.startswith('-') and digit [1:].isdigit() == True:
```

Comment:
Try to do this with one logical operation. Hint: look at `strip`, `lstrip`, `rstrip`

Reply:
if digit.lstrip('-').isdigit() == True: